### ISSUE TITLE
False positive unused flag

### ISSUE THREAD

**Issue by LievenDeFoor** [omecra/de.omecra.oedt.updatesite#62]
Hi,

The following construct incorrectly flags the procedure FindHandler as unused

```
DEFINE VARIABLE MyEditor AS HANDLE NO-UNDO.
DEFINE VARIABLE FrameHandle AS HANDLE NO-UNDO.

PROCEDURE CreateWidget:

    CREATE EDITOR MyEditor
        ASSIGN
            FRAME = FrameHandle
            NAME = "MyTestEditor":U
            TRIGGERS:
                ON "CTRL-F":U PERSISTENT
                    RUN FindHandler IN THIS-PROCEDURE.
            END TRIGGERS.
			
END PROCEDURE.

PROCEDURE FindHandler PRIVATE:

   MESSAGE "FindHandler" VIEW-AS ALERT-BOX.

END PROCEDURE.
```

**Comment by lneumeier** [comment]
Will be fixed in OEDT 9.1.0.

**Comment by lneumeier** [comment]
Fixed in 9.1.0
